packages/core/src/bag3d/core/assets/ahn/core.py

```diff
@@ -8,7 +8,7 @@
 
 class PartitionDefinitionAHN(StaticPartitionsDefinition):
     def __init__(self, ahn_version: int):
-        tile_ids = download_ahn_index_esri(ahn_version, with_geom=False)
+        tile_ids = download_ahn_index(with_geom=False)
```

Comment:
One note about checking the webservice in this __init__(): it will be ran at for each new job that depends on the ahn partition, so thousands of times in out pipeline. In the past this led to minor issues, ie. on rare cases the webservice is not reachable and it leads to a job error. 

Could we move this download function to an asset perhaps? ie. it downloads to a json and on this line we just read that json? that should fix the above issue. Or maybe there are other solutions.

Doesn't have to be part of this PR, but I think we should fix this at some point

Reply:
I see the issue, it is what is causing this when running dagster:

![image](https://github.com/user-attachments/assets/86856122-121e-4490-87cb-71bee80b8a62)

I am not familiar with Partition Definitions yet but I will look into this